### ISSUE TITLE
StackTrace when using paser.setTrace(True).

### ISSUE THREAD

**Issue by matclab** [antlr/antlr4-python3#15]
Unsing antlr 4.5-rc-2, and the antlr runtime for python3 from pipy, I obtain the following stack trace :
 ```
File "/home/clabaut/.virtualenvs/lll/lib/python3.4/site-packages/antlr4/Parser.py", line 381, in enterRule
    self.triggerEnterRuleEvent()
  File "/home/clabaut/.virtualenvs/lll/lib/python3.4/site-packages/antlr4/Parser.py", line 236, in triggerEnterRuleEvent
    listener.enterEveryRule(self._ctx)
TypeError: enterEveryRule() missing 1 required positional argument: 'ctx'
```

**Comment by ericvergnaud** [comment]
Fixed #16

**Comment by matclab** [comment]
Whow, that was fast.
Thanks !

-Mathieu

2015-01-19 16:12 GMT+01:00 ericvergnaud <notifications@github.com>:
